Makefile

```diff
@@ -197,14 +197,15 @@ scheduler-plugins-chart: scheduler-plugins-crd
 
 .PHONY: volcano-scheduler
 volcano-scheduler:
-	-@GOPATH=/tmp go install volcano.sh/volcano/cmd/scheduler@$(VOLCANO_SCHEDULER_VERSION)
+	rm -rf /tmp/volcano.sh/volcano
+	git clone -b $(VOLCANO_SCHEDULER_VERSION) --depth 1 https://github.com/volcano-sh/volcano /tmp/volcano.sh/volcano
```

Comment:
Due to https://github.com/golang/go/issues/44840, `go install` doesn't work fine.